pwa/src/templates/home/HomeTemplate.tsx

```diff
@@ -31,6 +32,18 @@ export const HomeTemplate: React.FC = () => {
     }
   };
 
+  const createTags = (dashboardCard: any): { label: string; tooltip: string }[] => {
+    const tagsArray: { label: string; tooltip: string }[] = [];
+
+    if (dashboardCard.object?.status) tagsArray.push({ label: dashboardCard.object.status, tooltip: "Status" });
+    if (dashboardCard.object?.lastCall)
+      tagsArray.push({ label: formatDateTime(t(i18n.language), dashboardCard.object.lastCall), tooltip: "Last call" });
+    if (dashboardCard.object?.lastRun)
+      tagsArray.push({ label: formatDateTime(t(i18n.language), dashboardCard.object.lastRun), tooltip: "Last run" });
+
+    return tagsArray;
+  };
```

Comment:
I think this logic should be in another file. Right now, the `HomeTemplate` holds some logic that is very "context heavy".

We could add it to the `DashboardCard.tsx` file, at the bottom, like so (dummy code):

```
const generateTagsFromDashboardCard = (dashboardCard: any): { label: string; tooltip: string }[] {
  ... logic
}
```

Afterwards, we can implement that function within the `HomeTemplate`, so instead of calling `createTags()` we can call `generateTagsFromDashboardCard`.

This way, the logic is written where the logic is relevant.